analyzers/tests/SonarAnalyzer.Test/TestCases/DeadStores.RoslynCfg.cs

```diff
@@ -1516,3 +1516,114 @@ public void TryReturns_Loop()
         private void SomethingThatCanThrow() { }
     }
 }
+
+public class PeachValidation
+{
+    // https://github.com/SonarSource/sonar-dotnet/issues/9466
+    public int ReadInFinallyAfterCatch()
+    {
+        var value = 0;
+        try
+        {
+            CanThrow();
+            value = 42;
+        }
+        catch
+        {
+            value = 1;  // Noncompliant FP, used in finally after the throw
+            throw;
+        }
+        finally
+        {
+            Log(value);
+        }
+        return value;
+    }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/9467
+    public int ReadAfterCatchAll_WithType(bool condition)
+    {
+        var value = 100;    // Noncompliant FP, used after catch all
+        try
+        {
+            CanThrow();
+            if (condition)
+            {
+                CanThrow();
+            }
+            value = 200;
+        }
+        catch (Exception exc)
+        {
+        }
```

Comment:
Slightly more ecological.
```suggestion
        catch (Exception ex) {  }
```

Reply:
While I'd normally do that, in this case, the catch has significant impact on the flow so I wanted to waste a bit of space on it.